modules/passkey/test/userstories/PasskeyCredentialCreation.spec.ts

```diff
@@ -1,8 +1,9 @@
 import { expect } from 'chai'
 import { deployments, ethers } from 'hardhat'
-import { WebAuthnCredentials, decodePublicKey, encodeWebAuthnSignature } from '../utils/webauthn'
+import { WebAuthnCredentials } from '../../test/utils/webauthn'
+import { decodePublicKey, encodeWebAuthnSignature } from '../../src/utils/webauthn'
 import { buildSignatureBytes } from '@safe-global/safe-4337/src/utils/execution'
-import { buildSafeTransaction, buildSafeTransactionData, SafeDomain } from '../utils/safe'
+import { buildSafeTransaction, buildSafeTransactionData, SafeDomain } from '../../src/utils/safe'
```

Comment:
The `safe` account implementation is only used for testing, I don't think it should move. Also, conceptually, tools for interacting with Safes should be implemented by the `safe-smart-account` package and not here.

Reply:
Moved back 👍🏾 

For using `safe-smart-account` (or `safe-contracts` older version), we don't have a `buildSafeTransactionData` in Safe Contracts I think on a quick look.